2023/DearAzure/data.json

```diff
@@ -0,0 +1,32 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "kasamshaikh",
+      "",
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Dear Azure",
+    "Country": "INDIA",
+    "Url": "https://www.eventbrite.com/o/dear-azure-az-india-by-kasam-shaikh-18178264307",
+    "Description_SessionizeApiId": "If you have configured your Sessionize correctly then browsing to this url (replacing {Your sessionize event Id} with your event sessionize id) will return valid json: https://sessionize.com/api/v2/{Your sessionize event Id}/view/Sessions",
+    "SessionizeApiId": "{Your sessionize event Id}",
```

Comment:
Will you be using Sessionize to show off your event sessions on globalazure.net?